### PR TITLE
Cpdp 1414 draft return saved content updates

### PR DESCRIPTION
**Pull request by AndyHWChung**

### CODE REVIEW

File: app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/ReturnSavedController.scala [hunk 1]
```diff
@@ -33,20 +33,20 @@ import uk.gov.hmrc.play.bootstrap.controller.FrontendController
 import scala.concurrent.{ExecutionContext, Future}
 
 @Singleton
-class ConfirmDraftReturnController @Inject() (
+class ReturnSavedController @Inject() (
```
Comment (mikail-khan) [inline app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/ReturnSavedController.scala]: what was the reason for the name change?

Comment (AndyHWChung) [reply app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/ReturnSavedController.scala]: the first one sounded funny - what's actually happening is we're telling the user that the draft return has been saved. "Confirming" a draft return doesn't quite make sense

Comment (mikail-khan) [reply app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/ReturnSavedController.scala]: could you rename this to `DraftReturnSavedController` pls?

File: conf/returns.routes [hunk 2]
```diff
@@ -21,4 +21,4 @@ GET   /pay-charge-in-return              uk.gov.hmrc.cgtpropertydisposalsfronten
 
 GET   /check-if-return-was-sent          uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.returns.CheckAllAnswersAndSubmitController.submissionError()
 POST  /check-if-return-was-sent          uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.returns.CheckAllAnswersAndSubmitController.submissionErrorSubmit()
-GET   /return-draft-confirmation         uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.returns.ConfirmDraftReturnController.confirmDraftReturn()
+GET   /exit/return-saved                 uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.returns.ReturnSavedController.draftReturnSaved()
```
Comment (mikail-khan) [inline conf/returns.routes]: change endpoint to `/exit/draft-return-saved`

Comment (AndyHWChung) [reply conf/returns.routes]: the url came from Jean - I'll leave it like that unless Jean wants it to be changed

File: app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/ReturnSavedController.scala (hunk 1 above)
Comment (AndyHWChung) [reply app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/ReturnSavedController.scala]: done

**Pull request merged**